example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -76,12 +78,9 @@ bool offb_ctrl_ned(Device &device)
 
     // Let yaw settle.
     offboard_log(offb_mode, " Let yaw settle...");
```

Comment:
This log is confusing - because at this point we haven't told the user we're yawing.  We also report done, which I think is redundant since we're not actually checking that we're done and we then say the next step.

Suggest replace with following text - states that we're yawing and gives user 2 seconds to watch that behaviour. We don't need to tell the watcher that yaw is settling, but we add a comment so programmers know why we added that sleep.
```
    offboard_log(offb_mode, " Turn (yaw) 90% clockwise...");
    device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 90.0f});
    sleep_for(seconds(2));  // Let yaw settle
```
Would also be cool if we could control the yaw rate - is that possible?

Reply:
makes sense